recipes/ptxcompiler/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "ptxcompiler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c7021a0946478404c4055a612fd725af95d53ec23eb3dd8d8bc0b5ac55eb44d0
+  fn: ptxcompiler.tar.gz
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version !=  "None")]
```

```suggestion
  skip: true  # [(not linux) or (cuda_compiler_version ==  "None")]
```

Should this be the condition?

---

Thank you for the review.
Without env variables build fails to get nvPTXCompiler.h.
The ptxcompiler requires cuda 11.1 or higher. How can i skip build for lower versions?